fineract-loan/src/main/java/org/apache/fineract/portfolio/loanproduct/api/LoanProductsApiResourceSwagger.java

```diff
@@ -663,6 +669,8 @@ private GetLoanProductsAccountingRule() {}
         public StringEnumOptionData capitalizedIncomeCalculationType;
         @Schema(example = "EQUAL_AMORTIZATION")
         public StringEnumOptionData capitalizedIncomeStrategy;
+        @Schema(example = "FEE")
+        public StringEnumOptionData capitalizedIncomeIncomeType;
```

Comment:
it can be `capitalizedIncomeType`. no need to `incomeIncomeType` ;)

Reply:
Yeah, i was kinda on autopilot when naming this, renamed it.